extensions/oidc/deployment/src/main/java/io/quarkus/oidc/deployment/devservices/keycloak/DevServicesConfig.java

```diff
@@ -97,57 +101,56 @@ public class DevServicesConfig {
     public boolean showLogs;
 
     /**
-     * Keycloak start command.
+     * Optional. Keycloak start command.
      * Use this property to experiment with Keycloak start options, see {@link https://www.keycloak.org/server/all-config}.
-     * Note it will be ignored when loading legacy Keycloak WildFly images.
+     * Note, it is ignored when loading legacy Keycloak WildFly images.
      */
     @ConfigItem
     public Optional<String> startCommand;
 
     /**
-     * The Keycloak realm name.
-     * This property will be used to create the realm if the realm file pointed to by the `realm-path` property does not exist,
-     * default value is `quarkus` in this case.
-     * If the realm file pointed to by the `realm-path` property exists then it is still recommended to set this property
-     * for Dev Services for Keycloak to avoid parsing the realm file to determine the realm name.
+     * Optional. The name of the Keycloak realm. The default value is `quarkus`.
```

Comment:
It feels like the message has somehow changed.
And right here, again, I think I'm not liking very much the `Optional.` prefix being added before even introducing the property name. I'd not mind it being reworked and making the text explain the optinality better

Reply:
The current text reads:
```
  /**
     * The name of the Keycloak realm.
     *
     * This property is used to create the realm if the realm file pointed to by the `realm-path` property does not exist.
     * The default value is `quarkus` in this case.
     * It's recommended to always set this property so that Dev Services for Keycloak can identify the realm name without
     * parsing the realm file.
     */
```